libnd4j/pom.xml

```diff
@@ -618,6 +620,46 @@
                 </plugins>
             </build>
         </profile>
+        <!-- FIXME: workaround to enable libnd4j tests -->
+        <profile>
+            <id>test</id>
+
+            <activation>
+                <property>
+                    <name>libnd4j.test</name>
+                </property>
+            </activation>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>1.6.0</version>
+
+                        <executions>
+                            <execution>
+                                <id>libnd4j.test.run</id>
+                                <!--FIXME: To run tests before build process-resources was chosen-->
```

Comment:
What happens at process-resources? Why do we need a profile?

Reply:
:smile: It's only workaround for now, to enable tests for libnd4j only and to run them before build, to not wait for a build itself, and get test results ASAP.

Reply:
Ok, we can probably use a property to set the phase instead. Let's do that instead of using a profile.

Reply:
Hm, ok let me try, I don't mind.